Sources/NIOEmbedded/AsyncTestingEventLoop.swift

```diff
@@ -57,7 +57,7 @@ import NIOConcurrencyHelpers
 /// `loop.run()`. As a result, if you need to await an `EventLoopFuture` created on this loop you
 /// should use `awaitFuture`.
```

Comment:
This is no longer true, right? Should we now deprecate `awaitFuture`?

Reply:
I think we can now just remove it.

Reply:
Oh, in fact this patch _does_ remove it: the comment is just vestigial.